src/error.rs

```diff
@@ -149,6 +149,8 @@ pub enum RoutingError {
     UntrustedMessage,
     /// Crypto related error.
     Crypto(safe_crypto::Error),
+    /// a new SectionInfo could not be pushed.
```

Comment:
The comment sounds like the error is not because the `New section info` is invalid, but ourself cannot accept it?